08-rails/common-view-helpers.md

```diff
@@ -45,6 +45,62 @@ The above `image_tag` will give you this HTML:
 ```html
 <img src="/assets/cat.jpg" alt="The cutest cat in the whole world.">
 ```
+
+## Using Named Routes
+
+Above we used hard-coded paths like "/profile" in our helpers.  We can also use methods like `link_to` with named routes from our `config/routes.rb` file.  
+
+For example:
+
+```erb
+<%= link_to "All Books", books_path %>
+```
+
+Can function as a link to the `books#index` action.  Rails automatically provides a helper for each path.  The name of each helper is the name of the path with `_path` on the end.
```

Comment:
We should probably be clear that Rails only provides route helper methods for each _named_ path.